### PR TITLE
Adds Python script to make pages from JSON files

### LINKED ISSUE #22: Make pages from combined JSON data
**Is your feature request related to a problem? Please describe.**
Possibly related to #21 , but we will need to add YAML frontmatter to JSON data before serving it up.

**Describe the solution you'd like**
Virtual vault can serve as a guide here, but we need to automatically generate yaml matter for JSON data. [Virtual Vault's makePages.py](https://github.com/RockefellerArchiveCenter/VirtualVault/blob/base/static_aid/makePages.py) can be used as an example.

### PR DESCRIPTION
**Pull request by helrond**
This currently writes only the title, id, layout and permalink to the front matter. VirtualVault (and staticAid) also write the description here, although that's really only for SEO reasons, which isn't really a concern for this site.

I've also added a layout for a single-item display which demonstrates how to fetch secondary objects like agents, subjects and top containers.

fixes #22

### CODE REVIEW

**Comment by helrond** [comment]
@p-galligan this is now ready for review.

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ rvm:
   - 2.6.3
 
 before_script:
- - chmod +x ./script/cibuild # or do this locally and commit
+  - which python
```
Comment (p-galligan) [inline .travis.yml]: Isn't this just identifying where python is installed? I understand the use for development, but is it serving another purpose?

**Review by p-galligan — COMMENTED** [review]
Just one small question about the travis file.

File: .travis.yml (hunk 1 above)
Comment (helrond) [reply .travis.yml]: Urgh, thought I deleted that. One sec.

**Review by p-galligan — APPROVED** [review]

**Pull request merged**